docs/man_pages/device/device-list-applications.md

```diff
@@ -1,24 +1,35 @@
 device list-applications
 ==========
 
-Usage | Syntax
+Usage | Synopsis
 ------|-------
 General | `$ appbuilder device list-applications [--device <Device ID>]`
-Lists the installed applications on all connected Android and iOS devices. 
-This operation is not applicable to Windows Phone devices.
 
-`<Device ID>` is the device index or identifier as listed by run `$ appbuilder device`
-Options:
-   * `--device` - If multiple devices are connected, sets the device for which you want to list all currently installed applications.
+Lists the installed applications on all connected Android <% if(isWindows || isMacOS) { %>and iOS <% } %>devices. 
+
+<% if(isConsole) { %>
+WARNING: You can work only with connected <% if(isWindows || isMacOS) { %>iOS and <% } %>Android devices.  
+<% if(isLinux) { %>WARNING: You cannot work with connected iOS devices on Linux systems. To view the complete help for this command, run `$ appbuilder help device list-applications`<% } %> 
```

Comment:
Actually, the two warnings are both redundant since we have explicit statements about the supported connected/virtual devices in the command description. I will drop them both.